lib/matplotlib/axes/_secondary_axes.py

```diff
@@ -1,30 +1,11 @@
-import collections
 import numpy as np
-import numbers
-
-import warnings
 
+import matplotlib.cbook as cbook
 import matplotlib.docstring as docstring
 import matplotlib.ticker as mticker
 import matplotlib.transforms as mtransforms
-import matplotlib.scale as mscale
-import matplotlib.cbook as cbook
-
 from matplotlib.axes._base import _AxesBase
-
-from matplotlib.ticker import (
-    AutoLocator,
-    AutoMinorLocator,
-    FixedLocator,
-    FuncFormatter,
-    LogFormatterSciNotation,
-    LogLocator,
-    NullLocator,
-    NullFormatter,
-    ScalarFormatter
-)
-
-from matplotlib.scale import Log10Transform
+from matplotlib.ticker import FixedLocator
```

Comment:
Why not leverage (the already imported) `mticker`? AFAICT, only line 273 would need to be updated from
```
self._axis.set_major_locator(FixedLocator(ticks))
```
to
```
self._axis.set_major_locator(mticker.FixedLocator(ticks))
```

Reply:
done